dashboard/app/helpers/lesson_import_helper.rb

```diff
@@ -0,0 +1,213 @@
+# Helper module for importing data from curriculumbuilder
+module LessonImportHelper
```

Comment:
I don't want to create a bunch of unnecessary work here, but I would caution against using rails helpers since they make it easy to violate encapsulation. If it is not too much work, I would recommend moving this file to `dashboard/lib/services/` and then requiring it directly . As another benefit, in the future we could move update_from_curriculum_builder into that file, excluding all of this code from our rails application entirely and only loading it into our import script.

Reply:
That sounds good to me. I'll move it over.